docs/portal/applications/using-advanced-ecp-features.md

```diff
@@ -0,0 +1,284 @@
+
+```yaml
+apiVersion: v1
+kind: LoadBalancer4 # ECP defined layer 4 load balancer.
+metadata:
+  name: demo-lb4 
+  namespace: demo # the kubernetes namespace created for your account
+spec: 
+  listeners:
+  - backendPort: 80 # port of backend pods to receive forwarded traffic
+    backendSelector: # select backend pods based on pod labels
+      app: demoapp
+    healthCheck: false # turn on/off health check. Set to false by default.
+    listenerPort: 80 # port exposed by the layer 4 load balancer. Must be the same as “backendPort”.
+    name: listener-80
+    protocol: tcp # can be “tcp” or “udp”
+    scheduler: lc # scheduling policy. Can be “lc”, or “rr”. Set to “lc” by default
+```
+
+## Load Balancer Health Checks
+
+You can configure TCP, UDP, and HTTP health checks for your Layer 4 load balancer.
+
+**TCP Health Check**
+
+```yaml
+apiVersion: v1
+kind: LoadBalancer4
+metadata:
+  name: demo-lb4 
+  namespace: demo
+spec: 
+  listeners:
+  - backendPort: 80
+    backendSelector:
+      app: demoapp
+    healthCheck: true
+    healthCheckType: tcp # can be “tcp” or “http”. “tcp” by default.
+    healthCheckConnectPort: 80 # same as backendPort by default
+    healthCheckTimeout: 10 # 10 by default. Scope: 1-50 in seconds
+    unhealthyThreshold: 3 # 3 by default. Must be a positive integer
+    healthCheckInterval: 1 # 1 by default. Scope: 1-5  in seconds
+    listenerPort: 80
+    name: listener-80
+    protocol: tcp
+    scheduler: lc
+```
+
+**UDP Health Check**
+
+```yaml
+apiVersion: v1
+kind: LoadBalancer4
+metadata:
+  name: demo-lb4 
+  namespace: demo
+spec: 
+  listeners:
+  - backendPort: 80
+    backendSelector:
+      app: demoapp
+    healthCheck: true
+    healthCheckType: udp  # “tcp” by default, please set it to “udp”.
+    healthCheckConnectPort: 80 # same as backendPort by default
+    healthCheckTimeout: 10 # 10 by default. Scope: 1-50 in seconds
+    unhealthyThreshold: 3 # 3 by default. Must be a positive integer
+    healthCheckInterval: 1 # 1 by default. Scope: 1-5  in seconds
+    listenerPort: 80
+    name: listener-80
+    protocol: udp # “healthCheckType” must be “udp” if the “protocol” here is “udp”
+    scheduler: lc
+```
+
+
+**HTTP Health Check**
+
+```yaml
+apiVersion: v1
+kind: LoadBalancer4
+metadata:
+  name: demo-lb4 
+  namespace: demo
+spec: 
+  listeners:
+  - backendPort: 80
+    backendSelector:
+      app: demoapp
+    healthCheck: true
+    healthCheckType: http # “tcp” by default. Please set it to “http”.
+    healthCheckDomain: "foo.bar.com"
+    healthCheckURI: "/index.html"
+    healthCheckConnectPort: 80 # same as backendPort by default
+    healthCheckTimeout: 10 # 10 by default. Scope: 1-50 in seconds
+    unhealthyThreshold: 3 # 3 by default. Must be a positive integer
+    healthCheckInterval: 1 # 1 by default. Scope: 1-5  in seconds
+    listenerPort: 80
+    name: listener-80
+    protocol: tcp
+    scheduler: lc
+```
+
+## Using Storage Resources
```

Comment:
Please help revise this section to remove anything related to persist-ssd. This storageclass has limited availability, and it is preferred not to show in documentation.

Reply:
Done. How should we handle other instances of "persist-ssd," such as the following bullet in the topic Understanding the Dashboard: Capacity of persist-SSD storage?

Reply:
Please remove "Capacity of persist-SSD storage" from "Understanding the Dashboard"

Reply:
Done.